stdlib/2/os/path.pyi

```diff
@@ -4,7 +4,8 @@
 # based on http://docs.python.org/3.2/library/os.path.html
 # adapted for 2.7 by Michal Pokorny
 
-from typing import overload, List, Any, Tuple, BinaryIO, TextIO, TypeVar, Callable, AnyStr
+from typing import overload, List, Any, Tuple, BinaryIO, TextIO, TypeVar, Callable, AnyStr, Union
+EitherStr = Union[bytes, unicode]
```

Comment:
You can't import this "module"; it is a stub.
`unicode` is Python 2 only and the syntax is Python 3 only, so it isn't executable code.
(And why wouldn't this apply to `List`, `Any`, etc?)